src/routing/core/anti_entropy.rs

```diff
@@ -43,41 +41,6 @@ pub(crate) fn process(
 
     let dst = msg.src().src_location().to_dst();
 
-    if let Ok(src_chain) = msg.proof_chain() {
-        if let Some(key) = network.key_by_name(&src_name) {
```

Comment:
do we need to check whether the incoming src_key is known to us?
and if unknown, shall we ask for an update?